api/_examples/cloud-accouts/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func main() {
+	// TODO @afiune maybe think about a way to inject CI credentials and
+	// run these examples as part of our CI pipelines
+	lacework, err := api.NewClient(os.Getenv("LW_ACCOUNT"),
+		api.WithSubaccount(os.Getenv("LW_SUBACCOUNT")),
+		api.WithApiKeys(os.Getenv("LW_API_KEY"), os.Getenv("LW_API_SECRET")),
+		api.WithApiV2(),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	res, err := lacework.V2.CloudAccounts.List()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, account := range res.Data {
+		support := "Unsupported"
+		switch account.Type {
+		case api.AwsCtSqsCloudAccount.String():
+			//case api.AwsCfgCloudAccount:
```

Comment:
Could we remove this commented out code?

Reply:
Just like my comment below, I will expect to uncomment this line, and other lines you have pointed out below in a couple of weeks. They are just examples of where we need to add new Cloud Accounts.

Though I am curious, do you think it is better to delete them and let the developer figure out where to add new Cloud Accounts? 🤔

Reply:
Yeah that's no problem as well. Guess for my part, it's been more so on our team, that, unless commented out code has had a reason for being there(or accompanying comment explaining why it's there), we've been stripping it out.

IMO (feel free to ignore) but I would suggest that there is a short comment left near any commented out code, just in case new people, or people outside the team happen to be checking out/working on the code. Wouldn't want it removed accidentally!

Reply:
@garyboyes I hear you! Thank you so much for your feedback ♥️